app/serializers/search/postgres/chat_channel_serializer.rb

```diff
@@ -0,0 +1,25 @@
+module Search
+  module Postgres
+    class ChatChannelSerializer < ApplicationSerializer
```

Comment:
I had to add this because most of the stuff that's returned in the ES "chat channel membership" is serialized at build time but we don't have that, so we need to serialize stuff in output. The format is different though